[#146274197] Instructions on configuring CDN header forwarding

henrytk opened the pull request:
## What

Instructions on configuring CDN header forwarding.

## How to review

Review in conjuction with the pull request to add this functionality on the paas-cdn-broker repository (hasn't been raised yet).

Proofread. Check the links work. See the repo's README for running instructions.

## Who can review

Anyone but me or @46bit

henrytk commented:
Hi @NeilBookless 

> May I suggest:
  "The Cloudfront domain can be found in the output from this command:
  [FIRST CODE SAMPLE}

I've pushed up a change to add this. I've put the code example inline though, because the paragraph is already followed by a code sample.

> Q: Why is there a delay? Something like

I've added a part to explain this is because it is a globally distributed network.

> Q: What is the significance of "all locations."? Are some more delayed than others? May be helpful to specify "for example" examples...

This should be addressed by the above point. It's not a latency issue, just the amount of time it takes AWS to provision a globally distributed network.

> Is 'hitting' a widely known technical term or just slang?

It's widely known, but not a technical term so I have changed it to 'sending requests to'

jollery commented on source/documentation/deploying_services/use_a_custom_domain.md:
```diff
@@ -105,7 +105,15 @@ After the record is created wait for up to an hour for the CloudFront distributi
 
 #### CDN Configuration Options
 
```
I wonder whether this section needs an intro to explain what the contents in the table is/relates to...

henrytk replied:
I agree. I've swapped the subheading for a short paragraph explaining what the table is for.

> ... and then a section here explaining how tenants should use the table to choose the right option for their needs. We should explicitly state here that more info on each can be found in the relevant sections below.

I haven't done this because the table doesn't help users do that, but added the reference to the subsections so they know what to read to get the useful information.

jollery commented on source/documentation/deploying_services/use_a_custom_domain.md:
```diff
@@ -105,7 +105,15 @@ After the record is created wait for up to an hour for the CloudFront distributi
+`domain` | *Required* | Your custom domain (or domains separated by commas) |
+`cookies` | *Optional* | Forward cookies to the origin | `true` |
+`headers` | *Optional* | A list of headers to forward to the origin | `["Host"]` |
 
```
... and then a section here explaining how tenants should use the table to choose the right option for their needs. We should explicitly state here that more info on each can be found in the relevant sections below.

henrytk commented:
@jollery I've addressed your comment, but rebased since so my response may not display in Github. Here's a link to it: https://github.com/alphagov/paas-tech-docs/pull/53#discussion_r131363893

dcarley commented:
I'll merge this when the deployment has reached production.
